### DISCUSSION TITLE
Flexible height for a header

### DISCUSSION THREAD

**Discussion by striveandsurvive** [gotenberg/gotenberg#917] [Q&A]
Hi,

Requirement is to have flexible height for the header of the PDF being generated. Content of the header is not same for all HTML being processed. Is there any way that can help to set the height of header according to its content so that it goes not get overlapped with the content of the body?

Thanks in advance.

**Comment by gulien** [comment]
The header/footer feature is a… feature from Chromium directly. It’s quite limited; it’s primary goal is to set page numbers etc.

I guess you have two options:

1. Set a heigh enough margin top
2. Create you own header with CSS and HTML on the page directly (quite harder IMO).
